src/Repositories/SentryRepository.cs

```diff
@@ -120,5 +120,28 @@ public async Task FinishSentry(string id, DateTimeOffset dateTime)
                 Console.Error.WriteLine(ex.ToString());
             }
         }
+
+        public async Task RegisterSentry(string id, DateTimeOffset dateTime)
+        {
+            try
+            {
+                var sentry = await GetSentryAsync(id);
+
+                var fb = Builders<Sentry>.Filter;
+                var filter = fb.And(
+                    fb.Eq(x => x.Id, id)
+                );
+                UpdateDefinition<Sentry> update = Builders<Sentry>.Update
+                    .Set(x => x.Registration, dateTime);
+                var res = await _dataContext.Sentries.UpdateOneAsync(filter, update);
+
+                await _logBookRepository.InsertLogBookEntryAsync($"Registered at ILS!", dateTime);
```

Comment:
Please use a localized string. 

- [ ] Add an entry to ` src/Resources/Repositories/*.resx` (You might want use the [ResX Editor](https://marketplace.visualstudio.com/items?itemName=DominicVonk.vscode-resx-editor))
- [ ] Replace the hard coded string with an reference to added entry
```c#
_localizer["<KEY>"]
```